### PR TITLE
add busco-prok with bacteria_odb

### LINKED ISSUE #848: [Container Request]: BUSCO lite with bacteria_odb
### What tool are you looking for and why?

A note to myself
 - prokaryote only (remove unnecessary stuff)
 - add bacteria_odb10 
 - naming? (slim, lite, prok, etc ..)

### PR DESCRIPTION
**Pull request by Kincekara**
Closes #848 
I left prokaryotes only tools to make a smaller image. 
Busco uses the working directory to search and download lineage sets. So I added bacteria linage into the data folder.

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by Kincekara** [comment]
@erinyoung, do you want any more changes to this image? I left this as a draft for now. 
I think we shouldn't write this to the latest tag.

File: busco/5.6.1-prok-bacteria_odb10_2024-01-08/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+CMD busco -h
+
+## Tests ##
+FROM app as test
+
+# offline test
+RUN wget -q https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/010/941/835/GCA_010941835.1_PDT000052640.3/GCA_010941835.1_PDT000052640.3_genomic.fna.gz  && \
+    gzip -d GCA_010941835.1_PDT000052640.3_genomic.fna.gz && \
+    busco --offline -l bacteria_odb10 -m genome -i GCA_010941835.1_PDT000052640.3_genomic.fna -o offline --cpu 4 && \
```
Comment (erinyoung) [inline busco/5.6.1-prok-bacteria_odb10_2024-01-08/Dockerfile:71]: I like that you include an offline test to make sure that it's working

**Comment by erinyoung** [comment]
I think this looks great! I don't have any changes to recommend, except for adding it to the main readme.

I think it's okay if the other, smaller version being the default 'latest' tag.

**Comment by Kincekara** [comment]
I couldn't find a name that is short and definitive, unfortunately. If you have something in your mind, we can change it.

**Comment by erinyoung** [comment]
Alright. I'm going to use that tag then.

You can check the status of the deploy at https://github.com/StaPH-B/docker-builds/actions/runs/7660600410

Thank you for putting this together!

**Pull request merged**